src/ol_infrastructure/applications/jupyterhub/Pulumi.applications.jupyterhub.CI.yaml

```diff
@@ -2,17 +2,33 @@
 secretsprovider: awskms://alias/infrastructure-secrets-ci
 encryptedkey: AQICAHi3MZ/Pjy2dahB1Qm+zKkKDPV1b9MYPGp7k649HPjmOHAFqnCOCAZSgyF1ZWf0UadNJAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMo8Vlyuwrp1ocOIPzAgEQgDtuHDo3PpUPnI8Ld39Bq5dR9m0mnxpqjeqfAWopoZssGhgjoG9CfYrqTNsw4qr8fHcnvtAHzJoKzyDD4w==
 config:
-  binderhub:domain: "binder.ci.learn.mit.edu"
-
+  jupyterhub:deployments:
+  - name: jupyterhub
+    domain: nb.ci.learn.mit.edu
+    namespace: jupyter
+    db_name: jupyterhub
+    apisix_ingress_class: apache-apisix
+    admin_users:
+    - "admin"
+    menu_override_file: menu_override.json
+    disabled_extension_file: disabled_extensions.json
+    extra_config_file: dynamicImageConfig.py
+  - name: jupyterhub-authoring
+    domain: authoring.nb.ci.learn.mit.edu
+    namespace: jupyter-authoring
+    db_name: jupyterhub_authoring
+    apisix_ingress_class: apache-apisix
+    admin_users:
+    - "admin"
+    menu_override_file: author_menu_override.json
+    disabled_extension_file: author_disabled_extensions.json
+    extra_config_file: dynamicImageConfig.py
```

Comment:
Note that I'll need to do this for all other envs too. Not hard, just something to do once it's functionally complete!